rotkehlchen/tests/unit/decoders/test_uniswapv4.py

```diff
@@ -330,3 +339,249 @@ def test_multi_pool_swap_to_second_address(
         counterparty=CPT_UNISWAP_V4,
         address=pool_address,
     )]
+
+
+@pytest.mark.vcr(filter_query_parameters=['apikey'])
+@pytest.mark.parametrize('polygon_pos_accounts', [['0x7E25f8488D25152437dBECC787F655966DD00C67']])
+def test_create_lp_position(
+        polygon_pos_inquirer: 'PolygonPOSInquirer',
+        polygon_pos_accounts: list['ChecksumEvmAddress'],
+) -> None:
+    tx_hash = deserialize_evm_tx_hash('0x2277ee48d4e5394a59500a9f70ab29c12b27b28398973e6ca2666143dd690358')  # noqa: E501
+    events, _ = get_decoded_events_of_transaction(evm_inquirer=polygon_pos_inquirer, tx_hash=tx_hash)  # noqa: E501
+    assert events == [EvmEvent(
+        tx_hash=tx_hash,
+        sequence_index=0,
+        timestamp=(timestamp := TimestampMS(1755608652000)),
+        location=Location.POLYGON_POS,
+        event_type=HistoryEventType.SPEND,
+        event_subtype=HistoryEventSubType.FEE,
+        asset=A_POLYGON_POS_MATIC,
+        amount=FVal(gas_amount := '0.012086010300538782'),
+        location_label=(user_address := polygon_pos_accounts[0]),
+        counterparty=CPT_GAS,
+        notes=f'Burn {gas_amount} POL for gas',
+    ), EvmEvent(
+        tx_hash=tx_hash,
+        sequence_index=446,
+        timestamp=timestamp,
+        location=Location.POLYGON_POS,
+        event_type=HistoryEventType.INFORMATIONAL,
+        event_subtype=HistoryEventSubType.APPROVE,
+        asset=(a_oauto := Asset('eip155:137/erc20:0x7f426F6Dc648e50464a0392E60E1BB465a67E9cf')),
+        amount=FVal(oauto_approval := '577.561778438973357269'),
+        location_label=user_address,
+        notes=f'Set oAUTO spending approval of {user_address} by 0x000000000022D473030F116dDEE9F6B43aC78BA3 to {oauto_approval}',  # noqa: E501
+        address=string_to_evm_address('0x000000000022D473030F116dDEE9F6B43aC78BA3'),
+    ), EvmEvent(
+        tx_hash=tx_hash,
+        sequence_index=448,
+        timestamp=timestamp,
+        location=Location.POLYGON_POS,
+        event_type=HistoryEventType.INFORMATIONAL,
+        event_subtype=HistoryEventSubType.APPROVE,
+        asset=(a_usdt := Asset('eip155:137/erc20:0xc2132D05D31c914a87C6611C10748AEb04B58e8F')),
+        amount=FVal(usdt_approval := '637.938815'),
+        location_label=user_address,
+        notes=f'Set USDT spending approval of {user_address} by 0x000000000022D473030F116dDEE9F6B43aC78BA3 to {usdt_approval}',  # noqa: E501
+        address=string_to_evm_address('0x000000000022D473030F116dDEE9F6B43aC78BA3'),
+    ), EvmEvent(
+        tx_hash=tx_hash,
+        sequence_index=449,
+        timestamp=timestamp,
+        location=Location.POLYGON_POS,
+        event_type=HistoryEventType.DEPOSIT,
+        event_subtype=HistoryEventSubType.DEPOSIT_FOR_WRAPPED,
+        asset=a_oauto,
+        amount=FVal(oauto_amount := '0.027630635454744533'),
+        location_label=user_address,
+        notes=f'Deposit {oauto_amount} oAUTO to Uniswap V4 LP',
```

Comment:
i think you can get the two tokens in the pool and add that to the notes.

`Deposit 4.55 oAURTO to Uniswap V4 oAUTO/USDT LP`